rpc/ethereum/backend/utils.go

```
@@ -147,7 +148,9 @@ func TxLogsFromEvents(codec codec.Codec, events []abci.Event) []*ethtypes.Log {
 			}
 
 			var log evmtypes.Log
-			codec.MustUnmarshal(attr.Value, &log)
+			if err := json.Unmarshal(attr.Value, &log); err != nil {
+				panic(err)
```

ah I wouldn't panic here, let's return an error

---

bump